dolphin/db/sqlalchemy/api.py

```diff
@@ -178,11 +181,11 @@ def access_info_update(context, access_info_id, values):
 
 def access_info_get(context, storage_id):
     """Get a storage access information."""
-    return _access_info_get(context, storage_id)
+    return _access_info_get(context, storage_id, models.AccessInfo)
```

Comment:
session is not required here?

Reply:
Actually, we don't have session in function parametsers, for now i don't know wether session is required or not. But for now, session are not being used.
 access_info_get(context, storage_id):  ==> no session parameter

Reply:
Ok :)